examples/example_3_transferHandling/example_3_transferHandling.h

```diff
@@ -28,7 +28,7 @@
 #include "QXmppClient.h"
 #include "QXmppTransferManager.h"
 
-class QBuffer;
+#include <QBuffer>
```

Comment:
On Monday 15 August 2016 15:29:28 Jeremy Lain� wrote:
> > @@ -28,7 +28,7 @@
> >  #include "QXmppClient.h"
> >  #include "QXmppTransferManager.h"
> >  
> > -class QBuffer;
> > +#include <QBuffer>
> 
> Why don't we use QT_FORWARD_DECLARE_CLASS here?

There are 2 ways to fix the issue with namespaced Qt:
1) use QT_FORWARD_DECLARE_CLASS when doing forward declares
2) include the class, since the Qt headers deal with the namespace (with the drawback of including a file)

I think in that case I ran my "find/sed" call only across the src directory but not across examples
and when compiling everything and hit the error, I just modified this example by hand, and since
it's just an example, it does not matter if I use approach 1 or 2

But in fact I found now that the QBuffer in this example header file
examples/example_3_transferHandling/example_3_transferHandling.h
is not needed at all.
So the simplest fix is: remove the "class QBuffer;" entirely

-- 
Best regards/Sch�ne Gr��e

Martin
A: Because it breaks the logical sequence of discussion
Q: Why is top posting bad?

()  ascii ribbon campaign - against html e-mail 
/\                        - against proprietary attachments

Geschenkideen, Accessoires, Seifen, Kulinarisches: www.lillehus.at